src/wwwroot/partials/settings/my-plan.html

```diff
@@ -7,21 +7,21 @@
   </div>
   <div class="section--wrapper my-plan--info-container">
     <div class="item flex flex--wrap first">
-      <div class="flex one flex--wrap">
+      <div class="flex one flex--wrap current-plan-price">
```

Comment:
In my opinion, HTML5 provides data attribute for cases like this. However, we are using empty or fake classes only for identifying an element. If we really have a class with css styles it ok, but it is not that case.